### DISCUSSION TITLE
Transfer ownership of a hobby team

### DISCUSSION THREAD

**Discussion by IvanRomero03** [vercel/community#3816] [Help] [answered]
### Summary

I need to transfer a team account ownership to another person that will lead the organization and I cannot do it inviting other member since the plan is the free tier

### Steps to Reproduce

I go to members to add the new owner and I cannot add it even tho I want to leave the current team since I wont be working on it
![image](https://github.com/vercel/community/assets/65189646/a2e2b00d-d661-4f79-b403-fa7d11078351)

I need to tranfer this team since I wont be working with this organization anymore

**Comment by swarnava** [comment, marked as answer]
Hobby team cannot be transferred unfortunately. They are meant for testing purpose and not for any production use. You need to create a new team and setup from scratch.
